core/database/foxx/api/tag_router.js

```diff
@@ -87,9 +141,29 @@ router
                     });
 
                     res.send(result);
+                    logger.logRequestSuccess({
+                        client: client?._id,
+                        correlationId: req.headers["x-correlation-id"],
+                        httpVerb: "POST",
+                        routePath: basePath + "/list/by_count",
+                        status: "Success",
+                        description: "List tags by count",
+                        extra: tot,
+                    });
                 },
             });
         } catch (e) {
+            logger.logRequestFailure({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "POST",
+                routePath: basePath + "/list/by_count",
+                status: "Failure",
+                description: "List tags by count",
+                extra: tot,
```

Comment:
Instead of just returning 

extra: tot

Which shows up in the logs like this

Nov 06 11:59:45 ci-datafed-arangodb arangod[1119]: 2025-11-06T16:59:45.353147Z [1119-7] INFO [99d80] {general} Client: unknown      | Correlation_ID: unknown | HTTP: POST           | Route: tag/list/by_count | Status: Success      | Desc: List tags by count | Extra: 1

Make sure instead of just saying extra, maybe include 

extra: { total_matching_tag: tot }